static/script/devices/mediaplayer/html5.js

```diff
@@ -587,7 +587,7 @@ require.def(
                         self._mediaElement.currentTime = self._sentinelSeekTime;
                     });
                 } else {
-                    this._sentinelSeekTime = currentTime;
+                    this._sentinelSeekTime = undefined;
```

Comment:
If we don't reset this member variable, then when time goes "backwards" by more than 15 seconds (the tolerance used by the seek sentinel), the seek sentinel activates. Although this isn't the bug we originally set out to fix, it seems just as much of a problem as the original issue, which was the buffering sentinel kicking in where time goes "backwards" by <15s.

An alternative solution could be to say that any device that makes time go "backwards" while playing live media cannot use the "seekable" live modifier, and we could have the other modifiers disable the seek sentinel.